indico/web/client/js/react/components/WTFPrivacyPolicyURLsField.jsx

```diff
@@ -0,0 +1,136 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2022 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import PropTypes from 'prop-types';
+import React, {useMemo, useState, useEffect} from 'react';
+
+import {Translate} from 'indico/react/i18n';
+
+export default function WTFPrivacyPolicyURLsField({fieldId, wrapperId, initialValues}) {
+  const parentElement = useMemo(() => document.getElementById(wrapperId), [wrapperId]);
```

Comment:
Probably not needed, you can just trigger the event on the hidden input itself (you can use a React ref to access it)